.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+sudo: required
```

Comment:
I wonder if this workaround for Chrome security sandboxing is still needed. We might first try pushing without this setting to see if it works.

Reply:
I was setting up Travis yesterday and ran into this; I'm guessing it is still needed.

Reply:
Yeah it's needed. https://travis-ci.org/github/image-crop-element/builds/396955361